### PR TITLE
Add application type endpoints for sources

### PR DESCRIPTION
**Pull request by ezr-ondrej**
We generated only sources endpoints, but we also need the application 
types.
This is adding that into the config and regenerates the client.

### CODE REVIEW

File: internal/clients/sources/client.gen.go [hunk 1]
```diff
@@ -2658,6 +3267,159 @@ func (c *ClientWithResponses) UnpauseSourceWithResponse(ctx context.Context, id
...
+		response.JSON404 = &dest
+
+	}
+
+	return response, nil
+}
+
 // ParseBulkCreateResponse parses an HTTP response from a BulkCreateWithResponse call
 func ParseBulkCreateResponse(rsp *http.Response) (*BulkCreateResponse, error) {
 	bodyBytes, err := ioutil.ReadAll(rsp.Body)
```
Comment (lzap) [inline internal/clients/sources/client.gen.go:3425]: Was this change intentional? Why you added bunch of methods?

Comment (ezr-ondrej) [reply internal/clients/sources/client.gen.go:3425]: It was, we have figured out we need more endpoints and we now only filter by enpoint tags, so we've added tag that has the endpoint required. I'd be happy to filter out what we do not use, but now we have just tags... so that is the control we use. We might end up dropping the `sources` tag in the end, but we are using it for now.
